source/api_reference/index.html.md.erb

```diff
@@ -3,54 +3,58 @@ title: API reference
 weight: 60
 ---
 
-# API reference
+# API reference 
 
-GOV.UK Pay is based on REST principles with endpoints returning data in JSON format, and standard HTTP error response codes. The platform API allows you to:
+The base URL for the GOV.UK Pay API is
+`https://publicapi.payments.service.gov.uk/`. 
 
-- initiate and complete payments
-- view the event history for individual payments
-- view transactions within a specified time period
-- provide full or partial refunds
+The same base URL is used for testing and live. The API key you use
+determines if the actions are treated as sandbox test payments, or processed as
+real payments.
 
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-## API overview
-
->The base URL for the API is __https://publicapi.payments.service.gov.uk/__
-
-The same base URL is now used for testing and live. The API key you use determines if the actions are treated as sandbox test payments or processed as real payments.
+The API is based on REST principles. It returns data in JSON format, and uses
+standard HTTP error response codes. 
 
-For full details of each API action, see the API Browser:
+For full details of each API action, see the API browser: 
```

Comment:
I'm a little unsure of using the term "API browser" and "API explorer" to mean the same service (Gelato).